app/build.gradle

```diff
@@ -35,6 +37,18 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
     }
+
+
+    ndkVersion '22.1.7171670'
```

Comment:
I have only one real question actually. Is this line a requirement for pre-installed NDK version or Gradle manages it and downloads NDK? Will Kotlin developers need to install NDK manually in order to build project locally?

We will move `arklib` into separate repo for usage in upcoming projects and only JNI wrapper will remain here, so ideally Kotlin developers would not know about Rust/NDK in this project at all.

Reply:
This NDK must be installed manually, as done in the _CI_ by using some _Github Action_. Take care of using the `22b` version when compiling the project locally with the _rustup_ commands specified in the _CI_ as well.

For sure, if this library is going to be used for other implementations, we could extract the different `.so` output files (4 files at `7MB` each one) and append them to this repository, so we delete the `./gradlew cargoBuild` _Gradle_ step and the _Rust_ source code from this repository.